lib/init/index.js

```diff
@@ -12,42 +13,45 @@ var languages = {
 var exportables = {};
 
 // Initialize the directory given the various options
-exportables.initProject = (opts) => {
-
+exportables.createNewProject = (options) => {
   // Set a default directory if one was not provided
-  opts.directory = opts.directory || path.resolve('.');
+  options.directory = options.directory || path.resolve('.');
 
   // Detect the requested language
-  var lang = exportables.resolveLanguage(opts);
+  var lang = exportables.resolveLanguage(options.lang);
 
   // If a language could not be detected
   if (lang === null) {
-    // Return an error
     return Promise.reject(new Error('Unrecognized language selection.'));
   } else {
     // Otherwise generate a project in that language
-    return lang.generateProject(opts);
+    return lang.generateProject(options);
   }
 };
 
 // Determine the langauge to initialize the project with
-exportables.resolveLanguage = (opts) => {
+exportables.resolveLanguage = (input) => {
```

Comment:
Deployment also has an  `exportables.resolveLanguage`, which accepts a string. I plan to refactor both to use the same functionality, so this is just a first step towards that.